.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
+on:
+  push:
+    branches: [ master ]
+  release:
+    types: [ published ]
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.8
+      - name: Coursier cache
+        uses: coursier/cache-action@v6
+      - name: sbt ci-release ${{ github.ref }}
+        run: ./sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
You will now need to add these secrets to GitHub -- I obviously can't have them.
You will find a helpful howto guide how to go about it at https://github.com/sbt/sbt-ci-release#gpg